content/en/agent/logs/_index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Agent Log collection
+kind: documentation
+description: Use the Datadog Agent to collect your logs and send them to Datadog
+further_reading:
+- link: "agent/logs/advanced_log_collection/#filter-logs"
+  tag: "Documentation"
+  text: "Filter logs send to Datadog"
+- link: "agent/logs/advanced_log_collection/#scrub-sensitive-data-in-your-logs"
+  tag: "Documentation"
+  text: "Scrub sensitive data in your logs"
+- link: "agent/logs/advanced_log_collection/#multi-line-aggregation"
+  tag: "Documentation"
+  text: "Multi-line Log aggregation"
+- link: "agent/logs/advanced_log_collection/#tail-multiple-directories-or-whole-directories-by-using-wildcards"
+  tag: "Documentation"
+  text: "Tail multiple directories or whole directories by using wildcards"
+- link: "agent/logs/advanced_log_collection/#global-processing-rules"
+  tag: "Documentation"
+  text: "Global processing rules"
+---
+
+Log collection requires an Agent version >= 6.0. Older versions of the Agent do not include the `Log collection` interface. If you are not using the Agent already, follow [the Agent installation instructions][1].
+
+Collecting logs is **disabled** by default in the Datadog Agent, you need to enable it in `datadog.yaml`:
```

Comment:
```suggestion
Collecting logs is **disabled** by default in the Datadog Agent. Enable log collection in the Agent's [main configuration file](/agent/guide/agent-configuration-files/#agent-main-configuration-file) (`datadog.yaml`):
```
Should we add tabs here for `Host Agent` and `Container Agent`. Then we can mention the environment variable `DD_LOGS_ENABLED`

Reply:
I'm not sure sine right below we link to the container Agent dedicated instructions whether it's pure Docker or K8